src/hibernate-resume/hibernate-resume-generator.c

```diff
@@ -93,7 +93,7 @@ static int process_resume(void) {
                 return log_error_errno(r, "Failed to generate unit name: %m");
 
         r = write_drop_in(arg_dest, device_unit, 40, "device-timeout",
-                          "# Automatically generated by systemd-cryptsetup-generator\n\n"
+                          "# Automatically generated by systemd-resume-generator\n\n"
```

Comment:
The program is called systemd-hibernate-resume-generator actually.

Reply:
Embarrassing! Fixed.